text/Cargo.toml

```diff
@@ -14,11 +14,12 @@ topological-sort = "0.2"
 regex.workspace = true
 chrono.workspace = true
 libc.workspace = true
+notify-debouncer-full = "0.3.1"
+ctor = "0.2.8"
```

Comment:
This project requires that dependencies are specified with only one or two decimals, e.g. "0.3" not "0.3.1" and "0.2" not "0.2.8"